### DISCUSSION TITLE
Kape.exe with _kape.cli in silent mode

### DISCUSSION THREAD

**Discussion by icaman** [EricZimmerman/KapeFiles#582] [Q&A] [answered]
Hi All,

I've been looking if someone successfully executed kape.exe using the CLI file in silent mode to avoid the user might kill the terminal by mistake losing the acquisition.
I don't see any flag in the list, and using the CLI kape launches a new process. 
I guess using a batch might be the best way to proceed.

Any help will be appreciated.

**Comment by EricZimmerman** [comment, marked as answer]
id think wrapping it in a batch file that doesnt spawn a window would do the trick, but ymmv

**Reply by icaman** [reply]
It worked! :) Thanks a mill for confirming that, Eric!
